package.json

```diff
@@ -48,21 +48,20 @@
   },
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk-ui#readme",
   "dependencies": {
-    "@forgerock/javascript-sdk": "^2.2.0"
+    "@forgerock/javascript-sdk": "../forgerock-javascript-sdk"
```

Comment:
Just temporarily adding this for easier testing. Will convert to official module before release.

Reply:
👍 I like this way... I was doing some silly things when I needed to test FRUI with latest (unreleased) JS-Core SDK... I'll put this in my release checklist so we don't forget it...